index.md

```diff
@@ -136,70 +137,70 @@ Committee Members:
+### Что произойдет, если кто-то нарушит Кодекс поведения?
 
-Our intent is that anyone in the community can stand up for this Code, and direct people who are unaware of this document. If that doesn't work, or if you need more help, you can contact [tc39-conduct-reports@googlegroups.com][tc39-conduct]. For more details please see our [Reporting Guidelines][].
+Мы стремимся к тому, чтобы любой член сообщества мог отстаивать этот Кодекс и направлять людей, не знакомых с этим документом<?>. Если это не сработает или вам нужна дополнительная помощь, вы можете связаться с [tc39-conduct-reports@googlegroups.com][tc39-conduct]. Для получения дополнительных сведений смотрите наши [рекомендации по оформлению жалоб[Reporting Guidelines].
 
-### Why do we need a Code of Conduct? Everyone knows not to be a jerk.
+### Зачем нужен Кодекс поведения? Все знают, что не нужно быть придурком.
 
-Sadly, not everyone knows this.
+К сожалению, не все это знают.
 
-However, even if everyone were kind, compassionate, and familiar with codes of conduct, it would still be incumbent upon our community to publish our own. Maintaining a Code of Conduct forces us to consider and articulate what kind of community we want to be, and serves as a constant reminder to put our best foot forward. But most importantly, it serves as a signpost to people looking to join our community that we feel these values are important.
+Однако, даже если бы все были добрыми, сострадательными и знакомыми с кодексами поведения, наше сообщество все равно обязалось бы опубликовать свой собственный кодекс. Соблюдение Кодекса поведения заставляет нас задуматься и сформулировать, каким сообществом мы хотим быть, и служит постоянным напоминанием о том, что мы должны делать для этого всё возможное. Но что более важно, это служит знаком для людей, желающих присоединиться к нашему сообществу, о том, что мы считаем эти ценности важными.
 
-### This is censorship! I have the right to say whatever I want!
+### Это цензура! Я имею право говорить все, что хочу!
 
-You do -- in _your_ space. If you'd like to hang out in _our_ spaces (as clarified above), we have some simple guidelines to follow. If you want to, for example, form a group where TC39 proposals are discussed using language inappropriate for general channels then nobody's stopping you. We respect your right to establish whatever codes of conduct you want in the spaces that belong to you. Please honor this Code of Conduct in our spaces.
+Имеете, в _вашем личном_ пространстве. Если вы хотите проводить время в _наших_ пространствах<?> (как указано выше), у нас есть несколько простых рекомендаций, которым нужно следовать. Если вы хотите, например, сформировать группу, в которой предложения TC39 обсуждаются с использованием языка, не подходящего для общих каналов, тогда вам никто не мешает. Мы уважаем ваше право устанавливать любые кодексы поведения в принадлежащих вам помещениях. Пожалуйста, соблюдайте этот Кодекс поведения на наших площадках.
```

Comment:
how to translate "spaces"?

Reply:
I think spaces should be translated, but dunno what the best translation would be. Definitely not a literal translation though, unless the russian word for "space" is usable in this context as well.

Reply:
Yeah, there is a similar meaning of space in Russian as in this case, I'm just not sure it fits well here. It sounds a bit awkward in Russian for "our spaces" in the meaning of both online and offline spaces. I wonder if it would be ok to use "our activities" instead